CHANGELOG.md

```diff
@@ -25,6 +25,9 @@ Dropping a requirement of a major version of a dependency is a new contract.
 
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian/jira-software-actions/compare/release-1.3.5...master
+[JPERF-836]: https://ecosystem.atlassian.net/browse/JPERF-836
+
+- Fix browse boards selector [JPERF-836]
```

Comment:
Nit: Issue key mention is inconsistent with the previous entries in the CHANGELOG. Not sure how much this matters.

Reply:
The inner peace is probably the thing for me :D

I'm fine with leaving it as it is.